docs/security.md

```diff
@@ -100,12 +118,23 @@ authorization:
   admin: 'lb_admin'
   user: 'lb_user'
   api: "lb_api"
-  ldapHost:
-  ldapPort:
-  ldapBindDn:
-  ldapSearch:
-  ldapPassword:
+  ldapConfigPath: "<ldap_config_path>"
+```
+
+The ldap config file should have the following contents
+
 ```
+  ldapHost: '<ldap sever>'
+  ldapPort: <port>
+  useTls: <true/false>
+  useSsl: <true/false>
+  ldapAdminBindDn: <>
+  ldapUserBaseDn: <>
+  ldapUserSearch: <>
+  ldapGroupMemberAttribute: <>
+  ldapAdminPassword: <>
```

Comment:
It would be good to have some description and examples of these properties for those less familiar with ldap. E.g. the code appears to assume `ldapUserSearch` is sth like `'(&(objectclass=user)(sAMAccountName=${USER}))`. 

`ldapAdminPassword` should be grouped with `ldapAdminBindDn`, and the doc should note they are the credentials of the service user that performs the ldap search. Many users may be familiar with Trino which doesn't require a service user.

Reply:
I agree that we need good documentation ldap and all the other security features like oauth2 and documentation in general. Let's do it in another PR.
My motivation behind this PR is to have a better solution than answering those repeated questions on the slack.